### ISSUE TITLE
WAR changedError

### ISSUE THREAD

**Issue by archiolochos** [nicoh88/homebridge-xiaomi-roborock-vacuum#93]
Hallo nicoh88,

i found an error in my logfile i dont know what to do with:
"WAR changedError | rockrobo.vacuum.v1 | Robot has an ERROR - charger-offline, Charger is offline"

what does that error mean? 
whats a WAD that changed? 
i have got a lot of this errors in my log - do i need to care?

**Comment by afharo** [comment]
Hi @archiolocho 
We are looking into this in #91 

Duplicate of #91

**Comment by afharo** [comment]
If you don’t mind I’ll close this one to have a bit of clean list of pending tasks

**Comment by nicoh88** [comment]
@archiolochos Is the charger unplugged? for sure?

@afharo For me, the charging station was really unplugged! But #91 still open.

**Comment by afharo** [comment]
> But problems in #91 still open.

Yes, I'll try to tackle it sometime this weekend :)

**Comment by nicoh88** [comment]
Fixed, waiting on version 0.6.6 or test the #master (0.6.6-dev) branch.

```
sudo su -
npm install -g nicoh88/homebridge-xiaomi-roborock-vacuum#master

```
